docs/topics/sam-with-receiver-plugin.md

```diff
@@ -34,7 +34,7 @@ does not have any built-in presets, and you need to specify your own list of spe
 
 ```groovy
 plugins {
-    id("org.jetbrains.kotlin.plugin.sam.with.receiver") version "$kotlin_version"
+    kotlin("plugin.sam.with.receiver") version "$kotlin_version"
```

Comment:
Unfortunately, this code only works for Kotlin DSL, not Groovy.

You can improve your PR by adding tabs, similar to the [all-open plugin](https://github.com/JetBrains/kotlin-web-site/blob/master/docs/topics/all-open-plugin.md#gradle) documentation. Include the short declaration for Kotlin, but retain the existing example for Groovy.

Reply:
Done

Which version placeholder is preferred? With underscore or without?

Reply:
Without,
Nice catch, there was a mistake in the code!